modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/wal/WalArchiveSizeConfigurationTest.java

```diff
@@ -103,6 +113,39 @@ public void testUnlimitedMaxArchiveSizeConfiguration() throws Exception {
         startGrid(0, (IgniteConfiguration cfg) -> cfg.setDataStorageConfiguration(dataStorageConfiguration));
     }
 
+    /**
+     * Checks that an exception is thrown if min WAL archive size is larger than max WAL archive size.
+     */
+    @Test
+    public void testIncorrectMinArchiveSizeConfiguration() {
+        DataStorageConfiguration dsCfg = new DataStorageConfiguration()
+            .setDefaultDataRegionConfiguration(new DataRegionConfiguration().setPersistenceEnabled(true))
+            .setMinWalArchiveSize(Long.MAX_VALUE);
+
+        assertThrowsAnyCause(
+            log,
+            () -> startGrid(0, (IgniteConfiguration cfg) -> cfg.setDataStorageConfiguration(dsCfg)),
+            IgniteCheckedException.class,
+            "DataRegionConfiguration.minWalArchiveSize must be less than or equal to"
```

Comment:
what?

Reply:
sorry, everything's fine here

Reply:
This is only for finding a substring of the error text